geopandas/tests/test_crs.py

```diff
@@ -295,9 +295,14 @@ def test_dataframe(self):
         assert df.geometry.values.crs == self.wgs
 
         # geometry column without geometry
-        df = GeoDataFrame({"geometry": [0, 1]})
-        df.crs = 27700
```

Comment:
Shouldn't this actually also raise a warning for setting the crs?

Reply:
It does, that is why I am checking that it does here. Maybe I don't understand the question?

Reply:
Not fully sure anymore ;), but _maybe_ I mean that both setting (`df.crs = 27700`) and accessing (`assert df.crs == self.osgb`) should raise a warning. 
But the current test would also pass if only one of both raises a warning.